code-samples/miscellaneous/parsing-mention-arguments/11/index.js

```diff
@@ -32,7 +32,7 @@ function getUserFromMentionRegEx(mention) {
 	return client.users.get(id);
 }
 
-client.on('message', message => {
+client.on('message', async message => {
```

Comment:
```suggestion
client.on('message', message => {
```

Reply:
The `async` is necessary for `await guild.ban(...)` to work.

Reply:
Completely missed that, oh well...